### ISSUE TITLE
Add more fun quotes to the help page

### ISSUE THREAD

**Issue by kylepollina** [kylepollina/objexplore#1]
https://github.com/kylepollina/objexplore/blob/main/objexplore/help_layout.py#L110

Need more fun quotes!

**Comment by npalladium** [comment]
How about: 
`I used objexplore and all I got was this lousy error message`?

PS. Great project! 🎉

**Comment by kylepollina** [comment]
Hahahaha, open a pull request! Become a contributor

**Comment by npalladium** [comment]
On a completely different note (don't want to create a new issue for this), could you please consider adding a more `enhancement` issues and a `hacktoberfest` label?

**Comment by kylepollina** [comment]
I'm sorry, I'm not sure what you are asking? Do you want some features to work on for hacktoberfest?

**Comment by npalladium** [comment]
Yes! And by `enhancement` issues I meant ones with code changes rather than data changes. 

I'm requesting for two separate things: 
1. Add more issues. 
2. Add `Hacktoberfest` label to the repo

Is this request clearer?

**Comment by kylepollina** [comment]
@npalladium I've added some more enhancements with a hacktoberfest label. Check 'em out!
